fe/fe-core/src/main/java/com/starrocks/common/proc/IndexSchemaProcNode.java

```diff
@@ -89,8 +89,8 @@ public ProcResult fetchResult() throws AnalysisException {
             String extraStr = StringUtils.join(extras, ",");
 
             List<String> rowList = Arrays.asList(column.getName(),
-                    column.getType().canonicalName(),
-                    column.isAllowNull() ? "Yes" : "No",
```

Comment:
Could you add a note that this must keep uppercase to avoid someone else change it back.